oracle/test/test.env

```diff
@@ -1,2 +1,3 @@
 HOME_RPC_URL=http://example.com
 FOREIGN_RPC_URL=http://example.com
+QUEUE_URL=http://example.com
```

Comment:
Where is this URL used?

Reply:
When importing the method `getRetrySequence` on the unit test, the following line runs and try to connect to the amqp broker so I had to add the url to avoid getting the error on the connection.

https://github.com/poanetwork/tokenbridge/blob/3be812d55f819765b355c044a4a6f4023a9a6af8/oracle/src/services/amqpClient.js#L2

Reply:
Thanks for explanation